docs/how-to/configure/titan.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Titan
+summary: Learn how to enable Titan in TiKV.
+category: how-to
+---
+
+# Titan
+
+Titan is an alternative key-value storage engine developed by PingCAP. It is based on RocksDB, but can reduce write amplification in RocksDB when using large values.
```

Comment:
```suggestion
Titan is an plugin of RocksDB developed by PingCAP to provide key-value separation, inspired by WiscKey. The goal of Titan is to reduce write amplification of RocksDB when using large values.
```

Reply:
A "storage engine" is a strong claim. A plugin of RocksDB is a more proper description of Titan.